tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -3648,4 +3648,17 @@ public function testDoWhileLoopInsideAnIfWithoutBrackets($expected, $input = nul
     {
         $this->doTest($expected, $input);
     }
+
+    public function testStructureSeparatedByTags()
+    {
+        $this->doTest(
+            '<?php for ($i = 0; $i < 20; $i++) {
+    ?>
+<?php
+} ?>',
+            '<?php for ($i = 0; $i < 20; $i++) {
+?>
+<?php 
```

Comment:
then, it will ultimatelly get trimmed by someone.

replace it with this trick:
```diff
-<?php 
+<?php '.'
```